### PR TITLE
Addresses Issue #6

### PR DESCRIPTION
**Pull request by dannydathua**
- Fixes Issue #6

### CODE REVIEW

**Comment by PolymorphiK** [comment]
I ran the tests and everything works well.

File: java/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+
```
Comment (PolymorphiK) [inline java/HelloWorld.java]: Remove whitespaces...

**Review by PolymorphiK — CHANGES_REQUESTED** [review]
See my comments/

**Review by PolymorphiK — APPROVED** [review]
Looks good!

**Comment by jasonw67** [comment]
Its good

**Comment by jasonw67** [comment]
It's good

**Pull request merged**